### ISSUE TITLE
Fix WP plugin list

### ISSUE THREAD

**Issue by patricktokeeffe** [washingtonstateuniversity/WSUWP-Plugin-Embeds#88]
Is this the repo for the **WSUWP Plugin Blocks** plugin?:

![image](https://user-images.githubusercontent.com/1348834/145491624-fd66137f-ca7e-49e9-81c5-8404b09de0d1.png)

If so

1. Will the plugin be updated to latest release? 
2. Could someone fix the 'visit site' link to the correct address?

**Comment by wsuwebsupport** [comment]
Hello Patrick,

Couple of quick questions for a little more info: 
Which site are you using the blocks plugin on? For most of our sites that should be off (there are some exceptions). The release version seems accurate to https://github.com/wsuwebteam/wsuwp-plugin-blocks?

**Comment by patricktokeeffe** [comment]
The site link for that plugin goes to <https://web.wsu.edu> so I had to guess which repo it might be. Looks like I guessed wrong oops. 

I was trying to figure out which plugin gave me access to the `wsuwp` shortcodes. My other guess, **WSUWP Tools**, has a [dead link](https://github.com/washingtonstateuniversity/wsuwp-plugin-tools):
![image](https://user-images.githubusercontent.com/1348834/145621707-2dd4bd4f-ae5a-47b2-8812-be2785587e91.png)

Maybe I misunderstood and the plugin isn't on the list. The wsuwp shortcodes are working fine. My site is labs.wsu.edu/lar
